deploy-0chain/action.yml

```diff
@@ -356,7 +356,7 @@ runs:
           sleep 10
           ((count++))
         done
-        if [[ $num -ne 6 ]]
+        if [[ $num -lt 6 ]]
```

Comment:
If the number of blobbers is too high, the faucet might not be enough. We are fauceting 12 tokens and staking 1 for each blobber. Would it be better to get $numBlobbers param from the user and replace `6` here and in the faucet loop with that?

Reply:
I get what you're saying but the number of blobbers is controlled by the helm charts so I don't want to add a param that 1. could be misleading and 2. means we have to update 7 repos 
If we start to deploy networks with > 6 blobbers frequently we can revisit this, maybe by counting the number of blobber pods or something